src/main/java/com/example/demo/batch/ReportBatch.java

```diff
@@ -249,11 +253,11 @@ private void createFilesTemporarily() throws IOException {
             // 1日の場合
             if (today == 1) {
                 try {
-                    File file = new File("src/main/resources/file/" + team.getName() + thisYear + "年" + thisMonth + "月.md");
+                    File file = new File("src/main/resources/file/" + msg.getMessage("file.name",new String[] { team.getName(),String.valueOf(thisYear),String.valueOf(thisMonth) }, Locale.JAPAN));
```

Comment:
ファイル名に関しては、msgよりはapplication.propertiesに持たせた方良いと思います。
また、このように分割して可読性を向上させてみてください。

```suggestion
                    // createFileNameの中でapplication.propertiesに記載されたファイル名のテンプレートを元に
                    // 特定の文字列({team_name}等)を置換させて、生成する。
                    String fileName =  createFileName();
                    File file = new File( "src/main/resources/file/" + fileName);
```

Reply:
修正しました。